distribution/kernel/pom.xml

```diff
@@ -131,7 +131,7 @@
                         <configuration>
                             <addTransitiveFeatures>false</addTransitiveFeatures>
                             <descriptors>
-                                <descriptor>file:${basedir}/target/classes/${project.artifactId}-features.xml</descriptor>
+                                <descriptor>mvn:${project.groupId}/${project.artifactId}/${project.version}/xml/features</descriptor>
```

Comment:
[KARAF-5311](https://issues.apache.org/jira/browse/KARAF-5311) NPE in karaf-maven-plugin when specifying descriptor by file url

Reply:
❗️ This will most likely cause problems when we release a new version and the artifact has never been deployed to Nexus yet.

Reply:
Shouldn't it be able to get it from the local repo where that new version has been built?

Reply:
I guess it depends _when_ this directive will be used. If it used at build time by the `karaf-maven-plugin`, it will probably fail because it runs as part of the `prepare-package` phase and the artifact hasn't been installed anywhere, not even in the local repo.

However, if the `<descriptors>` element is only used to add something that will be used at runtime then this should be fine.

Reply:
Based on the plugin description (https://svn.apache.org/repos/asf/karaf/site/production/manual/latest/karaf-maven-plugin.html):
> descriptors | String[] | The list of features XML repositories to use for the verify

So it sounds like this is used at build time. But we should still test it out to be sure.